.happy/terraform/modules/sfn/main.tf

```diff
@@ -174,6 +174,12 @@ resource "aws_sfn_state_machine" "state_machine" {
         "Type": "Task",
         "InputPath": "$",
         "Resource": "${var.lambda_error_handler}",
+        "Parameters": {
+          "execution.$": "$$.Execution.Id",
+          "error.$": "$.error",
+          "dataset_id.$": "$.dataset_id",
+          "collection_id.$": "$.collection_id"
+        },
```

Comment:
@Bento007 could you, for ease of review, add some clarity about that `$` and `$$` is doing here? I found [some documentation](https://states-language.net/) but I'm sorry to say it really only makes things like 10% more interpretable to me 😅